rust/lance-encoding/src/statistics.rs

```diff
@@ -165,6 +165,19 @@ impl GetStat for DataBlock {
     }
 }
 
+impl GetStat for FixedSizeListBlock {
+    fn get_stat(&self, stat: Stat) -> Option<Arc<dyn Array>> {
+        match stat {
+            Stat::MaxLength => {
+                let max_len = self.dimension * self.child.expect_single_stat::<UInt64Type>(stat);
```

Comment:
Does every block have `MaxLength`?  Is it possible the FSL's child doesn't have `MaxLength`?

Reply:
sorry, this is vestige, statistics for `fixedSizeList` data block is not supported in this PR.